src/test/java/com/truelayer/java/acceptance/PaymentsAcceptanceTests.java

```diff
@@ -186,6 +186,7 @@ public void shouldCompleteAnAuthorizationFlowForAPaymentWithPreselectedProvider(
         // assert that the link returned is good to be browsed
         URI bankPage = startAuthorizationFlowResponse
                 .getData()
+                .asAuthorizing()
```

Comment:
for extra clarity, this is technically not required but it will in future when we'll remove the `getAuthorizationFlow()` getter from the base abstract class.